src/main/java/net/imglib2/realtransform/DeformationFieldTransform.java

```diff
@@ -130,4 +135,43 @@ public RealTransform copy()
 	{
 		return new DeformationFieldTransform< >( this.defFieldReal );
 	}
+
+	/**
+	 * Converts a {@link RealTransform} into a deformation field.
+	 * 
+	 * Writes the result into the passed {@link RandomAccessibleInterval}. If
+	 * the transform has N source dimensions, then the deformation field must
+	 * have at least N+1 dimensions where the last dimensions of of length at
+	 * least N.  
+	 * 
+	 * A DeformationField creating with the resulting {@link RandomAccessibleInterval}
+	 * will give the same results as the transform inside its Interval.
+	 * 
+	 * @param transform
+	 *            the {@link RealTransform} to convert
+	 * @param deformationField
+	 *            the {@link RandomAccessibleInterval} into which the
+	 *            displacement field will be written
+	 */
+	public static < T extends RealType< T > > void fromRealTransform( final RealTransform transform, final RandomAccessibleInterval< T > deformationField )
+	{
+		assert deformationField.numDimensions() == ( transform.numSourceDimensions() + 1 );
+		assert deformationField.dimension( deformationField.numDimensions() - 1 ) >= transform.numSourceDimensions();
```

Comment:
Shouldn't the check be
```java
assert deformationField.dimension( deformationField.numDimensions() - 1 ) >= transform.numTargetDimensions()
```
instead of `numSourceDimensions`?

Reply:
Same for `int N = ....`

Reply:
Or is it always `numSourceDimensions == numTargetDimensions`? Then an assert for that equality would be good, I think.

Reply:
Yes, `DeformationFieldTransform`s can't handle the general case of arbitrary numSourceDimensions, and numTargetDimensions. And that's why I stuck with the check for equality.